typescript/nomad-sdk/src/nomad/messages/NomadMessage.ts

```diff
@@ -150,6 +150,7 @@ export class NomadMessage {
           messages.push(message);
         }
       } catch (e) {
+        console.log("An error occured while getting NomadMessage from Receipt", e);
```

Comment:
Might be useful to use `console.error` here instead so that it's more prominent in the console